content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Security Model
+description: Describes Istio's security model.
+weight: 10
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+This document aims to describe the security posture of Istio's various components, and how possible attacks can impact the system.
+
+## Components
+
+Istio comes with a variety of optional components that will be covered here.
+For a high level overview, see [Istio Architecture](/docs/ops/deployment/architecture/).
+Note that Istio deployments are highly flexible; below, we will primarily assume the worst case scenarios.
+
+### Istiod
+
+Istiod serves as the core control plane component of Istio, often serving the role of the [XDS serving component](/docs/concepts/traffic-management/) as well
+as the mesh [mTLS Certificate Authority](/docs/concepts/security/).
+
+Istiod is considered a highly privileged component, similar to that of the Kubernetes API server itself.
+* It has high Kubernetes RBAC privileges, typically including `Secret` read access and webhook write access.
+* When acting as the CA, it can provision arbitrary certificates.
+* When acting as the XDS control plane, it can program proxies to perform arbitrary behavior.
+
+As such, the security of the cluster is tightly coupled to the security of Istiod.
+Following [Kubernetes security best practices](https://kubernetes.io/docs/concepts/security/) around Istiod access is paramount.
+
+### Istio CNI plugin
+
+Istio can optionally be deployed with the [Istio CNI Plugin DaemonSet](docs/setup/additional-setup/cni/).
```

Comment:
```suggestion
Istio can optionally be deployed with the [Istio CNI Plugin `DaemonSet`](docs/setup/additional-setup/cni/).
```

Reply:
https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

Kubernetes doesn't use backticks for these kind of things.  Should we?

Reply:
We have across most of our docs when talking about Kubernetes resources